### DISCUSSION TITLE
Notifications to database not working

### DISCUSSION THREAD

**Discussion by Causeofhell** [filamentphp/filament#12847] [Help] [answered]
### Package

Notifications

### Package Version

v3.2

### How can we help you?

I have been trying to send notifications to database but is not working at all.
The table migration was executed and the table was created

1. I have set the databaseNotifications() in the adminpanelprovider 
2. i have test sending a notification by calling the beforeSave() 
3. since step number two is not working, i tried with an observer

any idea of why  is not working actually i am trying to send a basic notification 
Notification::make()
            ->title('Saved successfully')
            ->sendToDatabase(auth()->user());

**Comment by leandrocfe** [comment, marked as answer]
Are you using Laravel 11?
Did you run `php artisan queue:work` ?

**Reply by Causeofhell** [reply]
The command php artisan queue:work was the solutions, thanks for your time
